backend/tests/vscode-youi-events.spec.ts

```diff
@@ -265,37 +265,37 @@ describe('vscode-youi-events unit test', () => {
 	});
 
 	describe("doGeneratorDone", () => {
-		const createAndClose = "Create the project and close it for later use";
+		const createAndClose = "Create the project and close it for future use";
```

Comment:
Why not to use messages in the tests as well? In this way it will be no need to change tests next time.

Reply:
Still there's the package,json pref entry.
https://github.com/SAP/yeoman-ui/blob/chore-update_project_done_info_message/backend/package.json#L123

Reply:
Oh I see, so maybe const in describe?